packages/theme-get/README.md

```diff
@@ -0,0 +1,41 @@
+
+# @styled-system/theme-get
+
+The `themeGet` function is an existential getter function
+that can be used in any style declaration to get a value
+from your theme, with support for fallback values.
+This helps prevent errors from throwing when a theme value is missing,
+which can be helpful when unit testing styled-components.
+
+```sh
+npm i styled-system @styled-system/theme-get
```

Comment:
`themeGet` only uses the `get` utility from `@styled-system/core` which is a hard dependency, so you shouldn't need to install `styled-system` to use it

```suggestion
npm i @styled-system/theme-get
```